tests/integration/llm/client.py

```diff
@@ -831,6 +839,8 @@ def test_handler_rolling_batch(model, model_spec):
     seq_length = 100
     params = {"do_sample": True, "max_new_tokens": seq_length, "details": True}
     req["parameters"] = params
+    if "adapters" in spec:
+        req["adapters"] = spec.get("adapters")[0]
```

Comment:
This will use the same adapter for all requests, right? Do we want to test with multiple adapters across concurrent requests?

Reply:
I want to get this merged for now to unblock SAs, but I am planning to have an additional PR with some more testing later

Reply:
sounds good - i wanted to make sure i wasn't missing something